### PR TITLE
Optionally allow PasswordAuthentication in sshd_config

### PR DESCRIPTION
**Pull request by AMeng**
Finishing work started in https://github.com/johanmeiring/ansible-sftp/pull/2

### CODE REVIEW

File: templates/sshd_config.j2 [hunk 1]
```diff
@@ -6,4 +6,6 @@ Match Group {{ sftp_group_name }}
     AllowTCPForwarding no
     X11Forwarding no
     ForceCommand internal-sftp
+    {% if sftp_allow_passwords %}PasswordAuthentication yes{% endif %}
```
Comment (AMeng) [inline templates/sshd_config.j2]: @johanmeiring, I'm not sure if this is preferable over something like:

```
PasswordAuthentication {% if sftp_allow_passwords %}yes{% else %}no{% endif %}
```

In other words, should the fact that we default to `False` mean that we explicitly disable password authentication by default?

Comment (johanmeiring) [inline templates/sshd_config.j2]: @AMeng Thanks for the pull request :)  I think the explicit setting of `PasswordAuthentication` to `no` via the `else` statement makes a lot of sense and ensures that the correct behaviour is applied throughout.  Would be great if you could make that change please.

Comment (AMeng) [inline templates/sshd_config.j2]: @johanmeiring done.

**Comment by johanmeiring** [comment]
I like it :)  Thanks for the contribution.

**Comment by AMeng** [comment]
@johanmeiring would you mind creating a new release with these changes?

**Comment by johanmeiring** [comment]
@AMeng I've triggered a reimport after creating a new tag.  My guess is it should be ready soon. Will check tomorrow morning though...

**Comment by johanmeiring** [comment]
@AMeng Looks like 0.9.1 is now available on ansible-galaxy.  All good!

**Pull request merged**